README.md

```diff
@@ -99,6 +99,15 @@ yarn test
 yarn buildProd
 ```
 
+## Preparing a Release
+
+```sh
+VERSION=<VERSION>
+rm -r dist/
+yarn buildProd
```

Comment:
imo also include the command for installing dependencies

Reply:
Agreed. Done.